hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java

```diff
@@ -130,18 +133,18 @@ public int getSize() {
         return size;
     }
 
-    public CacheEvictionConfig setSize(int size) {
+    public EvictionConfig setSize(int size) {
```

Comment:
what about `size` smaller than 0? we can throw illegalArgumentException, or InvalidConfigurationException.